wordpress-mobile/WordPress-iOS
# Issues/reader card updates

Closes #5762 
Closes #5763
Closes #5769
Refs #6070

This PR updates the reader cards to match the new default cards being rolled out to the calypso reader.  The refreshed card styles are documented on an internal blog. Ping me for a link.  The changes supersede many of the requests in #5769 

To test:
Check the post list and confirm the card layout is correct and that there are no breaking constraints warnings in the console.
Confirm that follow, share, like, and comment buttons have text labels on wide screens (iPad, landscape iPhone) and do not have labels on small screens (portrait iPhone, multitasking iPad). 
Confirm that:
- the follow button toggles following status and reflects the correct status.
- the share button opens the share sheet
- the visit button opens the post in a browser
- the tag button continues to preview a tag. 
- the tab button highlights correctly when tapped

Note that for the tag button we're using the same trick as the blog name label and positioning a button with a larger tap area above the tag label to make it easier for the user to interact with it.

Additionally, the follow button in the tag and site headers is updated to match the new style.  Confirm these buttons continue to reflect the correct state. 

Needs review: @kurzee could I trouble you with this one?

cc @nbradbury @jancavan

### Examples
![simulator screen shot nov 9 2016 7 19 14 pm](https://cloud.githubusercontent.com/assets/1435271/20181211/dfce11fc-a722-11e6-9833-2a1789792150.png)

![simulator screen shot nov 9 2016 7 25 36 pm](https://cloud.githubusercontent.com/assets/1435271/20181230/ecf63c4c-a722-11e6-9c77-e5df8803b6aa.png)
